backend/EasyMeets.Core/EasyMeets.Core.WebAPI/appsettings.Development.json

```diff
@@ -4,5 +4,11 @@
       "Default": "Information",
       "Microsoft.AspNetCore": "Warning"
     }
+  },
+  "Jwt": {
+    "Firebase": {
+      "ValidIssuer": "https://securetoken.google.com/easymeets-4ed60",
+      "ValidAudience": "easymeets-4ed60"
+    }
```

Comment:
I believe we don't need these configurations here

Reply:
Fixed